tf2_bullet/include/tf2_bullet/tf2_bullet.h

```diff
@@ -32,12 +32,22 @@
 #define TF2_BULLET__TF2_BULLET_H_
 
 #include <tf2/convert.h>
+#include <LinearMath/btScalar.h>
 #include <LinearMath/btTransform.h>
 #include <geometry_msgs/msg/point_stamped.hpp>
 #include <tf2_ros/buffer_interface.h>
 
 #include <iostream>
 
+#if (BT_BULLET_VERSION == 282)
```

Comment:
Shouldn't the condition also be true for older versions?

Reply:
I didn't investigate older versions, as 2.82 is the oldest available on any platform that we support.

Reply:
I should be easy to check if older versions also lack the function and to update the condition to match the comment.

Reply:
Older versions do lack the function.  2.82 is the current oldest release available on their official website, but I updated to `<=` for completeness.